src/core/scene/a-scene.js

```diff
@@ -467,11 +512,29 @@ module.exports = registerElement('a-scene', {
     render: {
       value: function () {
         var effect = this.effect;
+        var camera = this.camera;
         var delta = this.clock.getDelta() * 1000;
         this.time = this.clock.elapsedTime * 1000;
+
         if (this.isPlaying) { this.tick(this.time, delta); }
+
         this.animationFrameID = effect.requestAnimationFrame(this.render);
-        effect.render(this.object3D, this.camera);
+
+        window.performance.mark('render-iteration-started');
```

Comment:
You mean the timings with performance.mark? I so, sure these should be done in a timing PR